examples/imagenet/configs/default.py

```diff
@@ -36,11 +36,14 @@ def get_config():
 
   # As defined in the `models` module.
   config.model = 'ResNet50'
+  # We use 'imagenette' in the Colab.
+  config.dataset = 'imagenet2012:5.*.*'
 
   config.learning_rate = 0.1
+  config.warmup_epochs = 5.0
   config.momentum = 0.9
   config.batch_size = 128
-  config.num_epochs = 100
+  config.num_epochs = 100.0
```

Comment:
Has this intentionally become a floating point number? Seems like it really should be an int? But if there was some good reasoning behind this then ignore me!

Reply:
It is intentional - it's fine to specify 2.5 epochs if you want. This is in parallel with the `warmup_epochs` (which in turn needs to be a float for the Colab's 0.5 warmup epochs).